Gruntfile.js

```diff
@@ -312,7 +312,7 @@ module.exports = function(grunt) {
         tasks: ['js']
       },
       content: {
-        files: ['**/*.html', '/downloads/*.md', '/guides/*.md', '/identity/*.md', '/ui-toolkit/*.md'],
+        files: ['index.html', '**/*.md'],
```

Comment:
Why are we even converting these? Let's let Jekyll handle the markdown files, just like we did on the CF docs site.